tests/models/decoder_only/vision_language/test_models.py

```diff
@@ -218,7 +218,6 @@
         auto_cls=AutoModelForVision2Seq,
         vllm_output_post_proc=model_utils.llava_video_vllm_to_hf_output,
         image_sizes=[((1669, 2560), (2560, 1669), (183, 488), (488, 183))],
-        runner_mm_key="videos",
         marks=[
             pytest.mark.skip(reason="LLava next video tests currently fail.")
         ],
```

Comment:
Is it intended that you just removed it for this model without adding it to options?